examples/test_nl.cc

```diff
@@ -137,131 +172,14 @@ int main() {
-  //   8,  8,  8,  8,  8,  8,  8,  8,  8,  8,  8;
-  // positions.transposeInPlace();
-
-  // // setting up the manager
-  // manager.update(positions, numbers, cell,
-  //                Eigen::Map<Eigen::Matrix<int, 3, 1>>{pbc.data()});
+  auto mat = features.get_feature_matrix();
 
-  // double cut_off{2};
-
-  // int mult{4};
-  // double rc_max{mult*0.5 + cut_off};
-  // // build a neighbourlist
-  // rascal::AdaptorNeighbourList<rascal::StructureManagerCenters>
-  //   pair_manager{manager, rc_max };
-  // // update to execute the build
-  // pair_manager.update();
-
-  // for (auto i{0}; i < mult; ++i) {
-  //   auto cutoff_tmp = i*0.5 + cut_off;
-  //   std::vector<std::vector<int>> neigh_ids;
-  //   std::vector<std::vector<double>> neigh_dist;
-
-  //   std::vector<std::vector<int>> neigh_ids_strict;
-  //   std::vector<std::vector<double>> neigh_dist_strict;
-
-  //   std::cout << "Setting up strict manager with rc="
-  // <<cutoff_tmp << std::endl;
-  //   // make strict neighbour list
-  //   rascal::AdaptorStrict<
-  //     rascal::AdaptorNeighbourList<rascal::StructureManagerCenters>>
-  //     adaptor_strict{pair_manager, cutoff_tmp};
-  //   // execute
-  //   adaptor_strict.update();
-
-  //   if (verbose) std::cout << "Setting get adaptor_strict info" << std::endl;
-  //   for (auto center : adaptor_strict) {
-  //     auto icenter{center.get_index()};
-  //     std::vector<int> indices_{};
-  //     std::vector<double> distances_{};
-
-  //     if (verbose) {
-  //       // get_index returns iteration index
-  //       std::cout << "strict atom out " << center.get_index();
-  //       // get_atom_index returns index from
-  //       std::cout << " " << center.get_atom_index() << " " ;
-
-  //       for (int ii{0};ii<3;++ii){
-  //         std::cout << center.get_position()[ii] << " ";
-  //       }
-  //       std::cout << " " << center.get_atom_type() << std::endl;
-  //     }
-
-  //     int Nneigh{0};
-  //     for (auto neigh : center) {
-  //       double distance{(center.get_position()
-  //                        - neigh.get_position()).norm()};
-  //       Nneigh += 1;
-  //       indices_.push_back(neigh.get_atom_index());
-  //       distances_.push_back(adaptor_strict.get_distance(neigh));
-
-  //       if (verbose) {
-  //         std::cout << "strict neigh out " << neigh.get_index();
-  //         std::cout << " " << neigh.get_atom_index() << "\t " ;
-
-  //         for (int ii{0};ii<3;++ii){
-  //           std::cout << neigh.get_position()[ii] << ", ";
-  //         }
-  //         std::cout << "\t dist=" << distance;
-  //         std::cout << "\t " << neigh.get_atom_type() << std::endl;
-  //       }
-
-  //     }
-
-  //     std::cout << "Number of Neighbourg: " << Nneigh << std::endl;
-  //     neigh_ids_strict.push_back(indices_);
-  //     neigh_dist_strict.push_back(distances_);
-  //     if (icenter > 1) break;
-  //   }
-  // }
+  for (size_t ii{0}; ii < mat.cols(); ii++) {
```

Comment:
this file is more of a playground to understand if something works rather than an example in this branch. Since it is still in progress I did not clean that up.